### DISCUSSION TITLE
Exporting to Meshlab

### DISCUSSION THREAD

**Discussion by 19kdc3** [marcomusy/vedo#782] [Q&A] [answered]
Hi Marco,

I'm trying to import a vedo mesh to meshlab (pymeshlab). I've tried using this line from your example, but `to_meshlab()` doesn't seem to exist. 
https://github.com/marcomusy/vedo/blob/cf356c051afb851ca5f8eca06d8bd710d057b59a/examples/other/pymeshlab1.py#L18

I've removed that function, and ended up with the following sample code:

```
def vedo_to_meshlab():
    model = vd.Mesh(vd.load("Sample Components\Cube.stl"))
    pml_mesh = utils.vedo2meshlab(model)
```

which has the error message: "normals() got an unexpected keyword argument 'compute'"

Would you happen to have any ideas or suggestions as to what's gone wrong?

Thanks for the help!

**Comment by marcomusy** [comment, marked as answer]
Hi thanks for reporting the issue.
I think I have now fixed both examples in the dev version:
```
pip install -U git+https://github.com/marcomusy/vedo.git
```
